doc/API/centreon-api-v2.yaml

```
@@ -1415,6 +1450,137 @@ components:
               type: integer
               description: "ID of the service on which the downtime is set"
               example : 5
+    Monitoring.ResourceMain:
+      type: object
+      properties:
+        type:
+          type: string
+          description: "Type of the resource"
+          example: "service"
+        id:
+          type: integer
+          format: int64
+          description: "ID of the resource"
+          example: 12
+        name:
+          type: string
+          description: "Resource name"
+          example: "Ping"
+        details_url:
+          type: string
+          description: "Resource url to details page"
+          example: "/monitoring/host/12/service/12"
+        icon:
+          type: object
+          properties:
+            name:
+              type: string
+              description: "Name of the icon"
+              example: "memory"
+            url:
```

url -> preview https://github.com/centreon/centreon/blob/master/src/Centreon/Domain/Entity/Image.php#L108
```suggestion
            preview:
```